executor/executable/controllabletask.go

```diff
@@ -387,17 +392,35 @@ func (t *ControllableTask) Kill() error {
 			"targetList": cmd.TargetList,
 		}).
 		Debug("state DONE not reached, about to commit transition")
+		
+		// Call cmd.Commit() asynchronous
+		commitDone := make(chan *CommitResponse)
+    	go func() {
+			var cr CommitResponse
+			cr.newState, cr.transitionError = cmd.Commit()
+			commitDone <- &cr
+		}()
+		
+		// Set timeout cause OCC is locking up so killing is not possible. The following approach
+		// help us to bypass the OCC transition, so we can kill the tasks on force destroy env. 
+		// Currently this will run for every Kill message receive (force or not).
+		// TODO: Find a better way to distinguish force from plain Kill message.
+		var _cr *CommitResponse
+		select {
+		case _cr = <- commitDone:
+		case <-time.After(45 * time.Second):
```

Comment:
Since these transitions are just the `Kill` path, let's set this to 15s. No process should take more than this, because we're not pushing values or reconfiguring, it's just cleanup.